pkg/occlient/volumes.go

```diff
@@ -14,7 +14,7 @@ import (
 
 // CreatePVC creates a PVC resource in the cluster with the given name, size and
 // labels
-func (c *Client) CreatePVC(name string, size string, labels map[string]string) (*corev1.PersistentVolumeClaim, error) {
+func (c *Client) CreatePVC(name string, size string, labels map[string]string, ownerReference ...metav1.OwnerReference) (*corev1.PersistentVolumeClaim, error) {
```

Comment:
please update the function comment with a description why there can be a multiple ownerReferences